labchecker/labcheck.py

```diff
@@ -1,25 +1,48 @@
 from plugins.base import Plugin
+from utils import _color, setnocolor
 import sys
 import getopt
+import yaml
 
 
 def usage():
-    print("Usage: labchecker [--type <plugin-type>] [--cloud <cloud-provider-name>]")
+    print("Usage: labchecker [--type <plugin-type>] [--cloud <cloud-provider-name>] [--nocolor]")
     print("optional arguments:")
     print("\t--help\t\tShow help message and exit.")
     print("\t--type\t\tSpecify a plugin type, like 'provider', 'all'(default)")
     print("\t--cloud\t\tSpecify a cloud provider, like 'otc', 'vexxhost', 'all'(default).")
+    print("\t--nocolor\t\tEnable the no color mode.")
     print("Exmaple:")
     print("labchecker check --type all --cloud all")
-    print("labchecker check --type provider --cloud vexxhost")
+    print("labchecker check --type nodepool --cloud vexxhost")
+    print("labchecker check --type nodepool --cloud vexxhost --nocolor")
+
+
+def get_cloud_list(cloud):
+    with open('/etc/openstack/clouds.yaml') as f:
+        clouds = yaml.load(f, Loader=yaml.FullLoader)
+        clouds_list = [c for c in clouds['clouds']]
+
+    if cloud not in clouds_list + ['all']:
+        print("Error: Cloud %s is not found." % cloud)
+        print("Please use the cloud in %s." % clouds_list)
+        exit(2)
+
+    clouds_list = clouds_list if cloud == 'all' else [cloud]
+    return clouds_list
+
+
+def header_print(header):
+    print(_color(header))
+    print(_color("=" * 48))
 
 
 def main(argv):
     cloud = 'all'
-    type = ''
+    ptype = ''
```

Comment:
@wangxiyuan type has been changed to ptype to avoid using the build-in keywords.

Reply:
cool